docs/src/rules/no-new-symbol.md

```diff
@@ -6,7 +6,7 @@ further_reading:
 - https://www.ecma-international.org/ecma-262/6.0/#sec-symbol-objects
 ---
 
-
+This rule was **deprecated** in ESLint v8.54.0 and replaced by the [no-new-native-nonconstructor](no-new-native-nonconstructor) rule.
```

Comment:
This should be _deprecated in ESLint v9.0.0_, right?

Reply:
Yes, v9.0.0.